test/Evolve.IntegrationTest.Cassandra/MigrationTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Cassandra.Data;
+using Evolve.Test.Utilities;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Evolve.IntegrationTest.Cassandra
+{
+    [Collection("Database collection")]
+    public class MigrationTest
+    {
+        private readonly CassandraFixture _cassandraFixture;
+        private readonly ITestOutputHelper _output;
```

Comment:
Nice, didn't know this one !

Reply:
It is good practice if you want to run tests in parallel (and it shows in the test output windows in VS).